lib/crud/students.ts

```diff
@@ -20,6 +20,8 @@ const validateIsOpportunityType = (value: string | undefined): value is Opportun
 
 export const updateStudent = studentOnlyAction(
   async (_: FormPassBackState, formData: FormData, studentId: string): Promise<FormPassBackState> => {
+    const isAdmin = (await auth())!.user.role === "ADMIN"
```

Comment:
use `Role.ADMIN` instead

Also, add a comment explaining the not null assertion

Reply:
Resolved in [ab865de](https://github.com/imperial/cpp-connect/pull/88/commits/ab865de7eac50722f9c8c42f0402f3e86edc048b)